morp/interface/__init__.py

```diff
@@ -77,6 +153,8 @@ def connect(self, connection_config=None):
         if self.connected: return self.connected
         if connection_config: self.connection_config = connection_config
 
+        self.connection_config.options.setdefault('max_timeout', 3600)
```

Comment:
Magic Number

Is there somewhere more visible that this could be defined as a constant?